### ISSUE TITLE
Subgrids: Rxbowtie not allowed

### ISSUE THREAD

**Issue by GGDRriedel** [gprMax/gprMax#436]
Beta again, I really need those subgrids :D

Error:
 #rx: {'p1': (1.582, 0.498, 1.5380000000000003), 'id': 'rxbowtie', 'outputs': 'Ey'} contains an output type that is not allowable. Allowable outputs in current context are ['Ex', 'Ey', 'Ez', 'Hx', 'Hy', 'Hz', 'Ix', 'Iy', 'Iz'].

In 
examples/subgrids/gssi_400_over_fractal_subsurface.py

Fll error stack: 
```
NoneType: None
Error creating user input object
Traceback (most recent call last):
  File "/media/raid0/Riedel/gprmaxv4/gprMax/gprMax/scene.py", line 70, in build_obj
    obj.build(grid, uip)
  File "/media/raid0/Riedel/gprmaxv4/gprMax/gprMax/cmds_multiuse.py", line 898, in build
    raise ValueError
ValueError
Traceback (most recent call last):
  File "/media/raid0/Riedel/gprmaxv4/gprMax/examples/subgrids/gssi_400_over_fractal_subsurface.py", line 158, in <module>
    gprMax.run(scenes=[scene], n=1, geometry_only=False, outputfile=fn, subgrid=True, autotranslate=True)
  File "/media/raid0/Riedel/gprmaxv4/gprMax/gprMax/gprMax.py", line 159, in run
    run_main(args)
  File "/media/raid0/Riedel/gprmaxv4/gprMax/gprMax/gprMax.py", line 219, in run_main
    results = context.run()
              ^^^^^^^^^^^^^
  File "/media/raid0/Riedel/gprmaxv4/gprMax/gprMax/contexts.py", line 83, in run
    model.build()
  File "/media/raid0/Riedel/gprmaxv4/gprMax/gprMax/model_build_run.py", line 74, in build
    self.reuse_geometry() if config.get_model_config().reuse_geometry else self.build_geometry()
                                                                           ^^^^^^^^^^^^^^^^^^^^^
  File "/media/raid0/Riedel/gprmaxv4/gprMax/gprMax/model_build_run.py", line 142, in build_geometry
    self.build_scene()
  File "/media/raid0/Riedel/gprmaxv4/gprMax/gprMax/model_build_run.py", line 268, in build_scene
    scene.create_internal_objects(self.G)
  File "/media/raid0/Riedel/gprmaxv4/gprMax/gprMax/scene.py", line 164, in create_internal_objects
    self.process_subgrid_cmds()
  File "/media/raid0/Riedel/gprmaxv4/gprMax/gprMax/scene.py", line 93, in process_subgrid_cmds
    self.process_cmds(sg_cmd.children_multiple, sg)
  File "/media/raid0/Riedel/gprmaxv4/gprMax/gprMax/scene.py", line 100, in process_cmds
    self.build_obj(obj, grid)
  File "/media/raid0/Riedel/gprmaxv4/gprMax/gprMax/scene.py", line 70, in build_obj
    obj.build(grid, uip)
  File "/media/raid0/Riedel/gprmaxv4/gprMax/gprMax/cmds_multiuse.py", line 898, in build
    raise ValueError
ValueError
```

latest Pull of the devel-Branch

**Comment by pyther-hub** [comment]
what is the exact thing causing this error if you could provide me some information I can work on this

**Comment by GGDRriedel** [comment]
I don't know. I am just running the provided example file

**Comment by craig-warren** [comment]
@GGDRriedel this has been created by trying to fix #431

**Comment by craig-warren** [comment]
@GGDRriedel this should now be fixed by https://github.com/gprMax/gprMax/commit/9f4bb79f54d47f040b49f88cbd9ad6630bfb2d4f We are now enforcing the use of a list when specifying even a single output component, e.g. `outputs = ["Ey"]` or `outputs = ["Ex", "Ey", "Ez"]`
